src/store/manager/Pagination.js

```diff
@@ -140,6 +137,22 @@ export default class Pagination {
 
 		try {
 			this.pageInfo = await this.fetchFunction(this.pageInfo, this.filterValue, this.store);
+
+			// Update isLastPage from REST endpoint
+			this.isLastPage = this.pageInfo.isLastPage;
+
+			// if isLastPage false from REST endpoint
+			// Do one more request to verify from REST
+			if (!this.pageInfo.isLastPage) {
+				const nextPage = {
+					...this.pageInfo,
+					pageNumber: this.pageInfo.pageNumber + 1
+				};
+
+				const { isLastPage } = await this.fetchFunction(nextPage, this.filterValue, this.store);
```

Comment:
What is a purpose of this?

Reply:
To check it's last page, if yes, it will return `[]`
because REST removed `isLastPage` status.

Reply:
Does it work?
Looks like it shoud be:
```
const { data: nextPageData } = await this.fetchFunction(nextPage, this.filterValue, this.store);
this.isLastPage = nextPageData.length === 0;
```

Reply:
Good catch!